docs/content/installation/upgrade-with-helm.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Upgrading NGINX Ingress Controller with Helm
+description: "This document describes how to use Helm to upgrade NGINX Ingress Controller from 2.x and 3.0.x to the latest available release."
+weight: 1800
+doctypes: [""]
+toc: true
+docs: "DOCS-603"
+---
+
+## Background
+
+In NGINX Ingress Controller version 3.1.0, [changes were introduced](https://github.com/nginxinc/kubernetes-ingress/pull/3606) to Helm resource names, labels and annotations to fit with Helm best practices.
+When using Helm to upgrade from a version prior to to 3.1.0, certain resources like Deployment, DaemonSet and Service will be recreated due to the aforementioned changes, which will result in downtime.
+Although the advisory is to update all resources in accordance with new naming convention, to avoid the downtime please follow the steps listed in this page.
```

Comment:
```suggestion
When using Helm to upgrade from a version prior to to 3.1.0, certain resources like Deployment, DaemonSet and Service will be recreated due to the aforementioned changes, which will result in downtime.

Although the advisory is to update all resources in accordance with new naming convention, to avoid the downtime please follow the steps listed in this page.
```

Really small change, but improves the readability by breaking up what would otherwise be a chunky paragraph of text into two separate texts - also draws more attention to the last line.